docs/interfaces.md

```diff
@@ -365,3 +365,10 @@ Auto-Connect: no
 Can manage timeservers directly separate from ``config core``.
 
 * Auto-Connect: no
+
+### tpm
+
+Can access the tpm device /dev/tpm0.
+
+Usage: reserved
+Auto-Connect: no
```

Comment:
Can you adjust this for the new format? FYI I notice that system-trace isn't following the proper format (feel free to fix it in passing if you want).